static-assets/components/cstudio-admin/mods/content-type-propsheet/image.js

```diff
@@ -117,67 +117,111 @@ YAHOO.extend(
-                    originalWidth <= widthConstrains &&
-                    heightConstrains &&
-                    originalHeight <= heightConstrains
-                  ) {
-                    var itemURL = to.fileName;
-                    _self.valueEl.value = itemURL;
-                    _self.value = itemURL;
-                    _self.updateFn(null, _self.valueEl);
-                  } else {
-                    var callback = {
-                      success: function (content) {
-                        var itemURL = content.message.internalName;
-                        _self.valueEl.value = itemURL;
-                        _self.value = itemURL;
-                        _self.updateFn(null, _self.valueEl);
+                const defPath = `/content-types/${CStudioAdminConsole.contentTypeSelected}/form-definition.xml`;
+                CrafterCMSNext.services.configuration
+                  .fetchConfigurationDOM(CStudioAuthoringContext.site, defPath, 'studio')
+                  .subscribe({
+                    next(doc) {
+                      let imageThumbnail = doc.querySelector('imageThumbnail');
+
+                      if (!imageThumbnail) {
+                        imageThumbnail = document.createElement('imageThumbnail');
+                        doc.appendChild(imageThumbnail);
                       }
-                    };
-
-                    CStudioAuthoring.Operations.cropperImage(
-                      CStudioAuthoringContext.site,
-                      message,
-                      imageData,
-                      widthConstrains,
-                      heightConstrains,
-                      widthConstrains / heightConstrains,
-                      null,
-                      callback
-                    );
-                  }
-                }
-                image.addEventListener('load', imageLoaded, false);
-                image.addEventListener('error', function (e) {
-                  message = CMgs.format(langBundle, 'loadImageError');
-                  CStudioAuthoring.Operations.showSimpleDialog(
-                    'error-dialog',
-                    CStudioAuthoring.Operations.simpleDialogTypeINFO,
-                    CMgs.format(langBundle, 'notification'),
-                    message,
-                    null, // use default button
-                    YAHOO.widget.SimpleDialog.ICON_BLOCK,
-                    'studioDialog'
-                  );
-                });
-
-                _self.getImage(url).subscribe((response) => {
-                  imageData.previewUrl = URL.createObjectURL(
-                    new Blob([response.response], { type: `image/${to.fileExtension}` })
-                  );
-                  image.src = imageData.previewUrl;
-                });
+                      imageThumbnail.innerHTML = to.fileName;
+
+                      const xmlFormDef = CrafterCMSNext.util.xml.beautify(CrafterCMSNext.util.xml.serialize(doc));
```

Comment:
Let's not beautify **and** just manually do the new XMLSerializer to serialize so we can remove the need for xml utils while we figure how to bundle it properly.

Reply:
(Also FYI, xmlUtils.serialize(...) beautifies already by default)